decisions/wc_united_sacrifice_decision.txt

```diff
@@ -0,0 +1,58 @@
+targetted_decisions = {
+	united_old_gods_sacrifice_prisoner_decision = {
+		only_playable = yes
+		filter = court
+		ai_target_filter = court
+		ai_check_interval = 12
+
+		diplomacy_icon = GFX_old_gods_sacrifice_decision
+
+		from_potential = { can_sacrifice_characters_to_og_potential_trigger = yes }
+
+		potential = {
+			THIS_is_prisoner_that_can_be_sacrificed_trigger = yes
+
+			host = { character = FROM }
+		}
+
+		allow = {
+			NOT = { diplomatic_immunity = yes }
+			FROM = { can_sacrifice_characters_allow_trigger = yes }
+		}
+
+		effect = {
+			FROM = {
+				show_scope_change = no
+
+				ROOT = {
+					show_scope_change = no
+
+					PREV_sacrifice_THIS_to_old_god_effect = yes
+				}
+			}
+		}
+
+		ai_will_do = {
+			factor = 1
+			modifier = {
+				factor = 0
+				FROM = { trait = cynical }
+			}
```

Comment:
I would remove or change it. The Old Gods give you power, so you can sacrifice people without zeal for your own interests.

Reply:
Can remove it, replace it with or add additional `content` trait requirement.
What will you pick?

Reply:
What if you're content, but zealous? :3

Reply:
Something like this?
```
modifier = {
	factor = 0
	FROM = {
		NOT = { trait = zealous }
		trait = content
	}
}
```